meshapp/src/components/password-forms/forgot-password-form.tsx

```diff
@@ -0,0 +1,103 @@
+import { useForm } from "react-hook-form";
+import { Button, Grid, Stack, Typography, TextField, Box } from '@mui/material';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import React, { useState } from "react";
+import { axiosInstance } from "../../config/axiosConfig";
+
+const theme = createTheme({
```

Comment:
You shouldn't be overwriting the existing theme for most of the changes in this case.
By overwriting the color, the dark theme will no longer work.
Instead of overriding the mui button it would make more sense to apply these changes using the sx prop.
Additionally making font changes here would make it more confusing and difficult to modify the theme in the future.